source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst

```diff
@@ -8,7 +8,52 @@ Drive Tab
 
 .. image:: images/using-the-labview-dashboard-with-c++-java-code/drive-tab.png
 
-Most of the indicators on the Drive tab utilize arrays.
+The :guilabel:`Select Autonomous...` dropdown can be used so show the available autonomous routines and choose one to run for the match.
+
+.. tabs::
+
+    .. code-tab:: java
+
+       SmartDashboard.putStringArray("Auto List", ["Drive Forwards", "Drive Backwards", "Shoot"]);
+
+       // At the beginning of auto
+       String autoName = SmartDashboard.getString("Auto Selector", "Drive Forwards") // This would make "Drive Forwards the default auto
+       switch(autoName) {
+          case "Drive Forwards":
+            // auto here
+          case "Drive Backwards":
+            // auto here
+          case "Shoot":
+            // auto here
+       }
+
+    .. code-tab:: c++
+
+       frc::SmartDashboard::PutNumber("Gyro", Drivetrain.GetHeading());
```

Comment:
These C++ and Java snippets should do the same thing but don't.

Reply:
Let me know if this looks good!